extensions/schema-compare/src/controllers/mainController.ts

```diff
@@ -7,7 +7,7 @@
 
 import * as azdata from 'azdata';
 import * as vscode from 'vscode';
-import { SchemaCompareDialog } from '../dialogs/schemaCompareDialog';
+import { SchemaCompareResult } from '../schemaCompareResult';
```

Comment:
seems like it is time to rename this to something like SchemaCompareMainWindow

Reply:
yes will rename soon in a separate PR